### PR TITLE
Fix Zone's Create/Update pages

### PR DESCRIPTION
**Pull request by TheMilek**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-admin-panel 
| Bug fix?        | no
| New feature?    | slighly
| BC breaks?      | no
| License         | MIT

Screenshot 2024-05-06 at 12 12 11

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Admin/ManagingZonesContext.php [hunk 1]
```diff
@@ -356,14 +356,14 @@ private function assertZoneAndItsMember(ZoneInterface $zone, ZoneMemberInterface
     }
 
     /**
-     * @Then /^I can not add a(?: country| province| zone) "([^"]+)"$/
+     * @Then /^I can not add a ([^"]+) "([^"]+)"$/
```
Comment (NoResponseMate) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingZonesContext.php]:
```suggestion
     * @Then /^I cannot add a ([^"]+) "([^"]+)"$/
```
though 
```suggestion
     * @Then /^I should not be able to add a ([^"]+) "([^"]+)"$/
```
would be better

File: src/Sylius/Behat/Context/Ui/Admin/ManagingZonesContext.php [hunk 2]
```diff
@@ -356,14 +356,14 @@ private function assertZoneAndItsMember(ZoneInterface $zone, ZoneMemberInterface
...
      */
-    public function iCanNotAddAZoneMember($name)
+    public function iCanNotAddAZoneMember(string $zoneType, string $name): void
```
Comment (NoResponseMate) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingZonesContext.php]: Do we even need to know the zone type in this instance?

File: features/admin/addressing/managing_zones/zone_validation.feature [hunk 3]
```diff
@@ -45,26 +45,26 @@ Feature: Zone validation
...
 
-    @todo @ui @api
+    @ui @api
     Scenario: Being unable to add itself to members during editing an existing zone
         Given the store has a zone "European Union" with code "EU"
         When I want to modify the zone named "European Union"
         Then I can not add a zone "European Union"
 
-    @todo @ui
+    @ui
```
Comment (NoResponseMate) [inline features/admin/addressing/managing_zones/zone_validation.feature]:
```suggestion
    @ui @todo-api
```
Just so we won't forget about it

File: features/admin/addressing/managing_zones/zone_validation.feature [hunk 4]
```diff
@@ -45,26 +45,26 @@ Feature: Zone validation
...
     Scenario: Seeing a disabled type field when adding country type zone
         When I want to create a new zone consisting of country
         Then I should not be able to edit its type
         And it should be of country type
 
-    @todo @ui
+    @ui
```
Comment (NoResponseMate) [inline features/admin/addressing/managing_zones/zone_validation.feature]:
```suggestion
    @ui @todo-api
```
same deal

File: src/Sylius/Behat/Element/Admin/Zone/FormElement.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            'code' => '[data-test-code]',
+            'form' => 'form',
+            'form_validation_message' => 'form > div.alert.alert-danger.d-block',
+            'name' => '[data-test-name]',
+            'scope' => '[data-test-scope]',
+            'type' => '[data-test-type]',
+            'zone_member' => '[data-test-zone-member]:contains("%name%")',
+            'zone_member_delete' => '[data-test-zone-member]:contains("%name%") button[name$="[delete]"]',
+            'zone_member_last' => '[data-test-members]:last-child',
+            'zone_member_added' => '[data-test-zone-member]:last-child option:not([selected="selected"])',
```
Comment (GSadee) [inline src/Sylius/Behat/Element/Admin/Zone/FormElement.php]: These keys are not in alphabetical order

File: src/Sylius/Bundle/AdminBundle/Resources/views/zone/form.html.twig
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/zone/form.html.twig:1]: Once this https://github.com/Sylius/Sylius/pull/16213 has been merged, move all templates to `templates` directory

**Review by GSadee — COMMENTED** [review]
In the PRs for admin panel, it would be nice to add some screenshots 🖖🏻

**Review by Wojdylak — APPROVED** [review]

**Review by mpysiak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]
But the rebase is needed

**Comment by Wojdylak** [comment]
Thank you, @TheMilek :tada:

**Pull request merged**